client/app/pages/queries/schema-browser.html

```diff
@@ -1,13 +1,21 @@
 <div class="schema-container">
-  <div class="p-t-5 p-b-5">
```

Comment:
Well, we needed some specific style declarations here, so I replaced two general classes with one more specific. We got a bit larger CSS, but lesser HTML :) I thought having just `schema-control` vs `p-t-5 p-b-5 schema-control` or `p-t-5 p-b-5 flex-container` will be a bit cleaner.

One of the biggest problems of CSS is that you constantly need to name things and all these names live in global namespace. So I totally understand your desire to keep CSS as little as possible :)

But there is a way to have many specific classes and keep CSS maintainable:
For example, we have `some-component`. All its styles goes to `some-component.css`. And this file have classes prefixed with component name. So when we see `.some-component` class name we know where to find its styles and we can be sure that changes in this file wouldn't affect other components.